src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -101,37 +101,33 @@ def is_safetensors_compatible(filenames, variant=None, passed_components=None) -
     - For models from the transformers library, the filename changes from "pytorch_model" to "model", and the ".bin"
       extension is replaced with ".safetensors"
     """
-    pt_filenames = []
```

Comment:
maybe we can use `model_folder_names` directly to simplifies a little bit https://github.com/huggingface/diffusers/blob/73acebb8cfbd1d2954cabe1af4185f9994e61917/src/diffusers/pipelines/pipeline_utils.py#L1357